packages/aws-cdk/README.md

```diff
@@ -560,6 +561,111 @@ This feature is currently in preview. Be aware of the following limitations:
+with the same resource configuration as the provided template.json.
+
+```console
+├── README.md
+├── bin
+│   └── my_awesome_application.ts
+├── cdk.json
+├── cdk.out
+│   ├── MyAwesomeApplication.assets.json
+│   ├── MyAwesomeApplication.template.json
+│   ├── cdk.out
+│   ├── manifest.json
+│   ├── synth.lock
+│   └── tree.json
+├── jest.config.js
+├── lib
+│   └── my_awesome_application-stack.ts
+```
+
+#### Generate a python application from a deployed stack
+
+If you already had a cloudformation stack deployed in your account and would like to migrate to using CDK instead of Cloudformation.
+You can use the `--from-stack` option to generate the application. In this case the `--stack-name` must match the name of the deployed stack. 
+
+```console
+$ # generate a python application from MyDeployedStack in your acount
+$ cdk bootstrap migrate --stack-name MyDeployedStack --language python --from-stack
+```
+
+This will generate a python CDK application which will synthesize to the same configuration of resources as the deployed stack.
+
+#### **CDK Migrate Limitations**
+
+CDK Migrate does not currently support nested stacks, Custom resources, or ForEach intrinsic function.
+CDK Migrate will only generate L1 constructs and does not currently support any higher level abstractions.
+CDK Migrate succesfully generating an application does *not* guarantee the application is immediately deployable.
+It simply generates a CDK application which will synthesize a template that has identical resource configurations 
+to the provided template. CDK Migrate does not interact with the CloudFormation service to verify the template 
+provided can deploy on it's own, nor does it validate that any resources in the provided template are already managed 
+in other CloudFormation Templates. In practice this is how CDK Migrate generated applications will 
+operate in the following scenarios:
+
+##### **The provided template is already deployed to cloudformation in the account/region**
+
+If the provided template came directly from a deployed cloudformation stack, and that stack has not experienced any drift, 
+then the generated application will be immediately deployable, and will not cause any changes to the deployed resources.
+Drift might occur if a resource in your template was modified outside of CloudFormation, namely via the console or CLI.
+
+##### **The provided template is not deployed to cloudformation in the account/region, and there *is not* overlap with with resources outside the template**
+
+If the provided template represents a set of resources that have no overlap with resources already deployed in the account/region, 
+then the generated application will be immediately deployable. A common reason for this might be because the template came 
+from a stack deployed in another account/region.
+
+In practice this means for any resource in the provided template. i.e.
```

Comment:
"i.e." means "that is", and "e.g." means "for example". I think it's better not to use those anyway.

```suggestion
In practice this means for any resource in the provided template, for example,
```